src/pydata_sphinx_theme/theme/pydata_sphinx_theme/theme.conf

```diff
@@ -18,6 +18,7 @@ twitter_url =
 icon_links_label = Icon Links
 icon_links =
 google_analytics_id =
```

Comment:
Can you define another `# DEPRECATE >= .12` section at the bottom of this file and move the old `google_analytics` config value there?

Reply:
Sure, I did not see you had that. It's a good idea to keep track of such things 👍